source/extensions/network/dns_resolver/getaddrinfo/getaddrinfo.cc

```diff
@@ -148,6 +150,7 @@ void GetAddrInfoDnsResolver::resolveThreadRoutine() {
     std::pair<ResolutionStatus, std::list<DnsResponse>> response;
     std::string details;
     {
+      next_query->addDetail("starting");
```

Comment:
this really heavyweight for something server side, where resolutions are happening at scale, but then anyone doing resolutions at scale really shouldn't be using this plug-in in the first place as it doesn't scale well.  I'm inclined to OK this but cc @mattklein123 for a second opinion